### ISSUE TITLE
error in process filter: Wrong type argument: hash-table-p, nil

### ISSUE THREAD

**Issue by SebastianCallh** [tomv564/pyls-mypy#25]
I installed `pyls-mypy` and now I get `error in process filter: lsp--on-diagnostics: Wrong type argument: hash-table-p, nil` when starting up and  `error in process filter: Wrong type argument: hash-table-p, nil` upon moving the cursor. Uninstalling `pyls-mypy` did not solve the problem. Any advice on this?

**Comment by tomv564** [comment]
Sounds like you're using emacs with lsp-mode ? Is there a way to get more logging from the mode so we can see what pyls-mypy may have sent that was wrong? 
You still have the problem now even though pyls-mypy is gone - perhaps you left some configuration for the mode that is causing the error?
I recommend asking the LSP client how to troubleshoot this issue as well, please fill me in on any problems pyls-mypy is causing!

**Comment by SebastianCallh** [comment]
Wow I didn't realize I opened the issue on the wrong repo. Thank you for you response. 
I nuked the virtual environment I was using, and never put any specifics in Emacs for `pyls-mypy` so I'm not sure if I was just not paying attention and this issue was there before as well. Since I still get the errors even without `pyls-mypy` I guess I shouldn't bother you any more :)  
I'll do some more digging!
